jetty-documentation/src/main/asciidoc/administration/startup/startup-base-vs-home.adoc

```diff
@@ -37,25 +37,23 @@ ____
 
 Potential configuration is resolved from these 2 directory locations.
 
-Check Jetty Base::
-  If the referenced configuration exists, relative to the defined Jetty base, use it.
-Check Jetty Home::
-  If the referenced configuration exists, relative to the defined Jetty home, use it.
+Check Jetty Base First::
+  If the referenced configuration exists, relative to the defined Jetty base, it is used.
+Check Jetty Home Second::
+  If the referenced configuration exists, relative to the defined Jetty home, it is used.
```

Comment:
There's also the intermediate jetty.dir configurations.

The --list-config also shows the search order in its output.